sanitize.css

```diff
@@ -0,0 +1,207 @@
+/* Box Model */
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
```

Comment:
What does this rule do? Why are u implementing it here? 🤔

Reply:
Sets the box-sizing property to border-box for elements :after of all elements  on the page.

Reply:
Sure, but the question here is... what does border-box do? How does it act over box model?

Reply:
border-box makes an element's width and height include padding and border, simplifying layout management and preventing overflow by ensuring total dimensions remain consistent. It contrasts with the default content-box.